CONTRIBUTING.md

```diff
@@ -22,4 +27,38 @@ code to fit with our [code style] or the project style, or discuss alternate
 ways of addressing the issue in question. When we're happy with everything,
 we'll bring your changes into master. Now you're a contributor!
 
+Installing Dependencies
+=======================
+
+On Debian/Ubuntu based systems
+------------------------------
+
+```
+sudo apt-get install -y ruby-dev libpq-dev libsqlite3-dev nodejs
+```
+
+Ubuntu, as of 14.04 ships with ruby 1.92. shoulda-matchers requires 2.x, so [install rvm], then install ruby-2.2:
```

Comment:
Is it possible to recommend rbenv in addition to RVM? Perhaps something like:

"shoulda-matchers requires Ruby 2.x, so use your Ruby version manager of choice to install the latest version of Ruby (2.2.1 at the time of this writing)."